code-generation/generator/src/main/resources/com/amazonaws/util/awsclientgenerator/velocity/cpp/ServiceClientSourceInit.vm

```diff
@@ -191,66 +154,6 @@ void ${className}::init(const ${clientConfigurationNamespace}::ClientConfigurati
 #if($metadata.hasEndpointTrait)
   m_enableHostPrefixInjection = config.enableHostPrefixInjection;
 #end
-#if($serviceNamespace == "S3Crt")
```

Comment:
If we remove this are we re-introducing the issue of s3-crt using two different authentication flows? are the generated client not included in this PR because it would be too big? we should maybe include those in the PR so that we catch any codegen issues before merging instead of at codegen time.

Reply:
> I tried to re-generate all clients and no diff were introduced.
 perfect, cool